rs/nervous_system/integration_tests/src/pocket_ic_helpers.rs

```diff
@@ -561,8 +592,7 @@ pub fn install_sns_directly_with_snsw_versions(
     }
 }
 
-#[track_caller]
```

Comment:
NCR

I assume this change is a tangential entrained change?

Reply:
Unfortunately track_caller is not compatible with async functions 😢